src/otx/cli/utils/help_formatter.py

```diff
@@ -0,0 +1,222 @@
+"""Custom Help Formatters for OTX CLI."""
+
+# Copyright (C) 2023 Intel Corporation
```

Comment:
and for the rest :)
```suggestion
# Copyright (C) 2024 Intel Corporation
```

Reply:
Thanks! - Done : https://github.com/openvinotoolkit/training_extensions/pull/2805/commits/3679a70296e06f1ed7c5fd6fab1899bf65fcf023